myosuite/envs/myo/walk_v0.py

```diff
@@ -134,4 +117,263 @@ def viewer_setup(self):
         self.viewer.cam.elevation = -15
         self.viewer.cam.distance = 5.0
         self.viewer.vopt.flags[3] = 1 # render actuators
-        self.sim.forward()
\ No newline at end of file
+        self.sim.forward()
+
+
+class WalkEnvV0(ReachEnvV0):
```

Comment:
@P-Schumacher  is there a reason this is inheriting from the ReachEnv? 

Seems like its overwriting everything. Maybe we can just inherit from the BaseV0?

Reply:
yes we can change it. I think just the viewer is still inherited